velox/row/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_executable(velox_row_benchmark UnsafeRowSerializeBenchmark.cpp)
```

Comment:
Please rename the folder to benchmarks to align with https://github.com/facebookincubator/velox/blob/main/velox/exec/benchmarks/CMakeLists.txt

Sorry, the previous comments is wrong, one executable should have one `main` function, so one benchmark one executable. The previous version is right.